models/models.go

```diff
@@ -47,7 +47,7 @@ type Filter struct {
 	LastUpdated     time.Time           `bson:"last_updated"               json:"-"`
 	ETag            string              `bson:"e_tag"                      json:"-"`
 
-	ID         string      `bson:"-"                    json:"id,omitempty"`
```

Comment:
this is probably fine, as long as we let the stakeholders beyond team b know that we are going this far

Reply:
@franmoore05 @eldeal :point_up_2: